docs/references/pysdk_api_reference.md

```diff
@@ -873,7 +873,8 @@ An `IndexInfo` structure contains three fields,`column_name`, `index_type`, and
   - Parameter settings for a full-text index:
     - `"ANALYZER"`: *Optional*
       - `"standard"`: (Default) The standard analyzer, segmented by token, lowercase processing, and provides stemming output. Use `-` to specify the languages stemmer. `English` is the default stemmer: `"standard-english"` and `"standard"` are the same stemmer setting. Supported language stemmers include: `Danish`, `Dutch`, `English`, `Finnish`, `French`, `German`, `Hungarian`, `Italian`, `Norwegian`, `Porter`, `Portuguese`, `Romanian`, `Russian`, `Spanish`, `Swedish`, and `Turkish`.
-      - `"chinese"`: Simplified Chinese
+      - `"rag"`: RAG analyzer imported from [RAGFlow](https://github.com/infiniflow/ragflow/blob/main/rag/nlp/rag_tokenizer.py). Multilingual support including `Chinese` and `English`. Use `-fine` to specify the fine-grained analyzer results.
```

Comment:
Multilingual RAG analyzer imported from [RAGFlow](https://github.com/infiniflow/ragflow/blob/main/rag/nlp/rag_tokenizer.py), supporting Chinese and English. Use `-fine` to specify the fine-grained analyzer results.